opendebates/views.py

```diff
@@ -186,6 +186,20 @@ def vote(request, id):
         voter.source = request.COOKIES['opendebates.source']
         voter.save()
 
+    if voter.user and voter.user != request.user:
```

Comment:
Me too! It would be nice to refactor the unregistered voting flow, but I'm guess that's in the 'nice-to-have' category.